modules/mbedtls/Kconfig.tls-generic

```diff
@@ -479,14 +479,22 @@ config MBEDTLS_SSL_EXTENDED_MASTER_SECRET
 	  which ensures that master secrets are different for every
 	  connection and every session.
 
+DT_CHOSEN_Z_ENTROPY := zephyr,entropy
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Ops, sorry, that's a leftover from the devlopment